### PR TITLE
:sparkles: New Tabs UI

### PR DESCRIPTION
**Pull request by KipandaJr**

### CODE REVIEW

File: src/lib/components/tabs/stories/index.stories.tsx [hunk 1]
```diff
@@ -5,14 +5,15 @@ import { Tabs } from '..';
...
-  //   },
-  //   info: {
-  //     control: { type: 'text' },
-  //   },
-  // },
+  argTypes: {
+    /*items: {
+      control: { type: 'array' },
+    },
+    type: {
```
Comment (okusindja) [inline src/lib/components/tabs/stories/index.stories.tsx:12]: Remove comments

Comment (okusindja) [reply src/lib/components/tabs/stories/index.stories.tsx:12]: @KipandaJr

Comment (KipandaJr) [reply src/lib/components/tabs/stories/index.stories.tsx:12]: Solved

File: src/lib/components/tabs/stories/index.stories.tsx [hunk 2]
```diff
@@ -5,14 +5,7 @@ import { Tabs } from '..';
 const meta: Meta<typeof Tabs> = {
   title: 'Tabs',
   component: Tabs,
-  // argTypes: {
-  //   items: {
-  //     control: { type: 'text' },
```
Comment (MarioBatalha) [inline src/lib/components/tabs/stories/index.stories.tsx:10]: How about this code smells?

File: src/lib/components/tabs/stories/index.stories.tsx [hunk 3]
```diff
@@ -5,14 +5,7 @@ import { Tabs } from '..';
...
-  //   },
-  //   info: {
-  //     control: { type: 'text' },
-  //   },
```
Comment (MarioBatalha) [inline src/lib/components/tabs/stories/index.stories.tsx:14]: Remove commented code

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — CHANGES_REQUESTED** [review]
Stories for Square Tabs

File: src/lib/components/tabs/stories/index.stories.tsx (hunk 3 above)
Comment (KipandaJr) [reply src/lib/components/tabs/stories/index.stories.tsx:14]: i just remove this.

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**